test/e2e/framework/assertions.go

```diff
@@ -311,6 +311,26 @@ func (f *Framework) AssertPrometheusExists(name, namespace string) func(t *testi
 	}
 }
 
+func (f *Framework) AssertPrometheusAlertmanagerEndpointsEmpty(name, namespace string) func(t *testing.T) {
+	return func(t *testing.T) {
+		err := Poll(time.Second, 5*time.Minute, func() error {
+			prom, err := f.MonitoringClient.Prometheuses(namespace).Get(ctx, name, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+
+			if len(prom.Spec.Alerting.Alertmanagers) != 0 {
```

Comment:
Can we check the number of discovered Alertmanager endpoints from the Prometheus config?

Reply:
Done!